totalopenstation/__init__.py

```diff
@@ -1,3 +1,13 @@
 # -*- coding: utf-8 -*-
 
 __version__  = '0.5.dev'
+
+import logging
+from pprint import pformat
+
+def geo_to_debug(data):
```

Comment:
I see your intentions with this auxiliary function but the pprint output doesn't play very well with line-formatted log output, see this example: [tops-debug.log](https://github.com/steko/totalopenstation/files/3719314/tops-debug.log).
Furthermore, not all input formats include this debug function, I would prefer to leave it out for now and add it later separately.

Reply:
Ok for later use.
BTW, for long feature such the one in __init__.py, it is more redeable than a huge line. I opened it with Notepad++ maybe an other viewer as not the same output.

Reply:
Removed function.